app/templates/views/pricing/index.html

```diff
@@ -28,29 +28,29 @@ <h1 class="heading-large">Pricing</h1>
 
   <p class="govuk-body">
     {% if not current_user.is_authenticated %}
-      <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.register') }}">Create an account</a> then add as many unique services as you need to.</p>
+      <a class="govuk-link govuk-link--no-visited-state" href="{{ url_for('main.register') }}">Create an account,</a> then add your unique service(s).</p>
```

Comment:
[question] should we call these "services"? Or something more like "Program"?

Reply:
I left it as services for continuity throughout the rest of the content. I suppose we could go through and change all the times the word appears, but I think for now, `service` is fine.

Ideally, we do a lot more content overhaul once we have some staff devoted to it.